pipeline/vr/appendvrids.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+ARTIFACT_DIR="$1"
+INPUT_FILE="$2"
+OUTPUT_FILE="$3"
+IMAGE_NAME="${4:-brcachallenge/append-vr-ids:0.1}"
+SEQ_REPO_DIR="${5:-/usr/local/share/seqrepo}"
+
+# --- pre-step: build vr image (caching will make this fast if it was already built)
+docker build -t "${IMAGE_NAME}" .
+
+# ----------------------------------------------------
+# --- 1. spin up seqrepo-rest-service
+# ----------------------------------------------------
+docker run --rm -d \
```

Comment:
this should probably be done via the `start-seqrepo-rest-service`

Reply:
testing keeping this here so it can run in isolation...
if it fails because one's already running, that seems harmless